src/pages/home/ReportScreen.js

```diff
@@ -66,8 +66,11 @@ const propTypes = {
     /** The report metadata loading states */
     reportMetadata: reportMetadataPropTypes,
 
-    /** Array of report actions for this report */
-    reportActions: PropTypes.arrayOf(PropTypes.shape(reportActionPropTypes)),
+    /** All the report actions for this report */
+    reportActions: PropTypes.objectOf(PropTypes.shape(reportActionPropTypes)),
```

Comment:
I noticed this was set to be an array, but I checked the data in Onyx (it's an object) and the place that uses this data also has it typed in TS as a `Record` (which would be an object and not an array).